### DISCUSSION TITLE
False diversion

### DISCUSSION THREAD

**Discussion by rickpilot601** [osmandapp/OsmAnd#19337] [Maps Related]
This problem has been the subject iof numerous emails. I finally got into github. I have the latest map of Thailand, 1 March 2024, latest app version. Southbound on hwy 2 from Nong Khai to Udon Thani, it makes a diversion, going north it does not. Unfortunately videos have to be less than 10mb,ridiculous. 

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/163732422/945952e4-3b04-442a-ae2c-b26c6037278f"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/163732422/187eeeec-e9e3-4a46-8aa6-b3e9e779829d"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/163732422/47e4321b-0107-464b-81af-275329511641"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by Sonwon1** [comment]
You may want to switch over to brouter, it does a better job routing in my opinion. You will find it in the manual.

**Comment by DmitryAlexei** [comment]
Please indicate:
- the starting and end points of the route, 
- what version of the app do you use
- we also need your profile settings to reproduce this issue: go to menu > settings > your driving profile > export profile. The file can be attached to github by adding .txt extention.
